libs/coin-modules/coin-aptos/src/logic/lastBlock.ts

```diff
@@ -0,0 +1,7 @@
+import type { BlockInfo } from "@ledgerhq/coin-framework/api/index";
+import { AptosAPI } from "../network";
+
+export async function lastBlock(): Promise<BlockInfo> {
```

Comment:
Ideally we want to use it also in the sync method but it looks like you get it via getAccountInfo, just fyi